HISTORY.md.license

```diff
@@ -0,0 +1,4 @@
+SPDX-FileCopyrightText: 2020 Helmholtz Centre for Environmental Research (UFZ)
+SPDX-FileCopyrightText: 2020 Helmholtz-Zentrum Dresden-Rossendorf (HZDR)
```

Comment:
Dear Tobias @tobiashuste, would you like to also change the copyright year to 2022 here as well?

```suggestion
SPDX-FileCopyrightText: 2022 Helmholtz Centre for Environmental Research (UFZ)
SPDX-FileCopyrightText: 2022 Helmholtz-Zentrum Dresden-Rossendorf (HZDR)
```

Reply:
@christianhueserhzdr This is changed now.